index.bs

```diff
@@ -1736,240 +1736,207 @@ whether they are exceptions, promise rejection values, or properties.
 
-These can be functionality provided by the underlying operating system, or provided by a native
-third-party library to interact with a device. These are an abstraction which "wrap" the
-native functionality without introducing significant complexity while securing the API surface
-to the browser, hence are called wrapper APIs.
+These can be functionality provided by the underlying operating system,
+or provided by a native third-party library
+to interact with a device.
+These are an abstraction which "wrap" the native functionality
+without introducing significant complexity,
+while securing the API surface to the browser.
+So, these are called wrapper APIs.
 
 This section contains principles for consideration when designing APIs for devices.
 
 <h3 id="device-ids">Use care when exposing identifying information about devices</h3>
 
-Merely exposing the presence of a device
-may harm user privacy.
-Exposing
-additional information about a device
-or its capabilities
-increases this risk further.
-Exposing device identifiers
-increases this risk even more.
-Any such information
-increases the set of fingerprinting data available to sites.
-But there are possible user privacy harms here
-that go beyond fingerprinting.
-Please read [[FINGERPRINTING-GUIDANCE]] and [[UNSANCTIONED-TRACKING]]
-for additional details.
-
-Think carefully about
-whether it is really necessary
-to expose identifying information about the device at all.
-Please consider if your use cases could be satisfied
-by a less powerful API. [[LEAST-POWER]]
 
-Despite these concerns,
-you may find that it is necessary
-to expose identifying information about a device to the web platform.
-
-The following guidelines will help ensure
-that this is done in a consistent and privacy-friendly way:
-
-: Limit identifiable information in the id
-:: As much as possible,
-    device ids exposed to the web platform
-    should not contain identifiable information
-    such as branding, make and model numbers, etc.
-    In many cases
-    using a randomized number or unique id
-    is preferable to
-    a generic string identifier such as "device1".
-
-    Device ids expressed as numbers
-    should contain sufficient entropy
-    so as to avoid re-use or potential sharing among other devices,
-    and should not be easily guessable.
+If you need to give web sites access to information about a device,
+use the guidelines below to decide what information to expose.
+
+Firstly, think carefully about whether it is really necessary
+to expose identifying information about the device at all.
+Consider whether your [user needs](#priority-of-constituencies) could be satisfied
+by a less powerful API.
+
+Exposing the presence of a device,
+additional information about a device,
+or device identifiers,
+each increase the risk of harming the user's privacy.
+
+One risk is that as more specific information is shared,
+the set of
+[fingerprinting data](https://www.w3.org/TR/fingerprinting-guidance/)
```

Comment:
I prefer using Bikeshed ref syntax (`[[FOO]]`) so that the references section gets properly filled out. This can be instead of or in addition to Markdown links like this one.

Reply:
Oh, I see you have these at the end of the section, Okay.